README.md

```diff
@@ -80,6 +80,7 @@ Every argument is optional.
 | [ignore-updates](#ignore-updates)                                   | Any update (update/comment) can reset the stale idle time on the issues/PRs | `false`               |
 | [ignore-issue-updates](#ignore-issue-updates)                       | Override [ignore-updates](#ignore-updates) for issues only                  |                       |
 | [ignore-pr-updates](#ignore-pr-updates)                             | Override [ignore-updates](#ignore-updates) for PRs only                     |                       |
+| [close-as-not-planned](#close-as-not-planned)                       | Use the "not planned" close reason for issues                               |                       |
```

Comment:
IMO because this only affects issues (PR closed state is either unmerged or merged), and because of the existence of options like `close-issue-message`, `close-issue-label`, etc. This option should be `close-issue-<something>`.

Also, right now it's a boolean (either not planned or completed), but that can change if GitHub ever introduces a 3rd closed state. That's why I suggested in https://github.com/actions/stale/issues/744#issue-1251957794 to have a `close-issue-reason` option that takes a string.

But don't make any changes just for me :slightly_smiling_face: 
Let's wait and see what the maintainers think.

Reply:
Would you want to validate against a known set of values, or just pass it through to the API verbatim?

Reply:
If I pass an invalid reason, I get 422 Unprocessable Entity:

```json
{
  "message": "Please specify a valid state reason.",
  "documentation_url": "https://docs.github.com/rest/reference/issues#update-an-issue"
}
```

Reply:
Sounds good, it just means that if GitHub introduces a new value in the future, we'll have to push another update to this action.

Reply:
@luketomlinson changes made, please take another look when you get a chance. Thanks!